src/tidyversetopandas/tidyversetopandas.py

```diff
@@ -1,13 +1,12 @@
 import pandas as pd
 
-def arrange(df: pd.DataFrame, *col_name:str):
+def arrange(df: pd.DataFrame, ascending: bool = True, *col_name:str):
     """
     Sort a Pandas dataframe in the ascending order
 
-    This function takes a Pandas dataframe and names of the columns, according to which the dataframe is sorted ascendingly
+    This function takes a Pandas dataframe and names of the columns, according to which the dataframe is sorted ascendingly/descendingly
 
     Parameters:
-    -------
     - df(pandas.DataFrame): The input dataframe object.
     - *col_name(string)
 
```

Comment:
you can add description for the new flag
```suggestion
- ascending(bool): Specifies the sorting order. Set to True for ascending order, or False for descending order. Default is ascending order.
```